apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/Playground/index.tsx

```diff
@@ -45,7 +45,7 @@ export default function Playground({
   evaluation: EvaluationDto
 }) {
   const { findEvaluation } = useEvaluations()
-  const evaluation = findEvaluation(serverEvaluation.uuid)!
+  const evaluation = findEvaluation(serverEvaluation)
```

Comment:
Why not using the original `serverEvaluation` though?

Reply:
Because if you use the `serverEvaluation` is not reacting to changes when the prompt is saved.